symengine/cwrapper.cpp

```diff
@@ -200,4 +210,78 @@ int is_a_Symbol(const basic c)
     return is_a<Symbol>(*(*RCP_const_cast(c)));
 }
 
+
+// C wrapper for std::vector<int>
+
+struct CVectorInt {
+    std::vector<int> m;
+};
+
+CVectorInt* vectorint_new()
+{
+    return new CVectorInt;
+}
+
+int vectorint_placement_new(CVectorInt *self, size_t size)
+{
+    if (!SymEngine::is_aligned(self)) return 2;
+    if (size < sizeof(CVectorInt)) return 1;
+    new(self) CVectorInt;
+    return 0;
+}
+
+void vectorint_free(CVectorInt *self)
+{
+    delete self;
+}
```

Comment:
vectorint_placement_free() is now implemented. I checked with Valgrind that there are no memory leaks.